data/json/mapgen/lab/lab_surface/lab_surface_nested.json

```diff
@@ -1910,23 +1863,13 @@
     "type": "mapgen",
     "method": "json",
     "nested_mapgen_id": "surface_sub_hall_clutter",
-    "object": {
-      "mapgensize": [ 1, 1 ],
-      "rows": [ "X" ],
-      "terrain": { "X": "t_null" },
-      "furniture": { "X": [ "f_IV_pole", "f_desk", "f_counter" ] }
-    }
+    "object": { "mapgensize": [ 1, 1 ], "rows": [ "X" ], "furniture": { "X": [ "f_IV_pole", "f_desk", "f_counter" ] } }
```

Comment:
While the null is removed, it seems to have been moved to a single line

Reply:
That's just the JSON style rules changing the formatting.